### PR TITLE
website link formate, field label center align issue

### PR DESCRIPTION
**Pull request by aliyazahid**
website link label formated ossue fixed also. other field label were not center align when i format it from html editor. this issue is also fixed

### CODE REVIEW

File: package.json
Comment (Kiho) [inline package.json:1]: Why do we need new dependency inflect here? Can you remove it and revert package-lock.json

File: src/form-elements/index.jsx
Comment (Kiho) [inline src/form-elements/index.jsx:1]: You can't have duplicated className attribute

File: src/form-elements/index.jsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ class TextInput extends React.Component {
       <div style={{ ...this.props.style }} className={baseClasses}>
         <ComponentHeader {...this.props} />
         <div className="form-group">
-          <ComponentLabel {...this.props} />
+          <ComponentLabel {...this.props}  className={'form-label'}/>
```
Comment (Kiho) [inline src/form-elements/index.jsx:113]: I don't think adding className to every ComponentLable is good idea.
Can you make change here https://github.com/Kiho/react-form-builder/blob/master/src/form-elements/component-label.jsx#LL8C5-L8C46 to 
```js
<label className={props.className || 'form-label'}>
```

**Comment by aliyazahid** [comment]
Ok. I thought you are calling prop.className may be it reflecting some
other component thats why i didn't call tge className directly. No problem
i will change it.

**Comment by aliyazahid** [comment]
HI Kiho
I updated the code as you said. now can you please approve my PR
https://github.com/blackjk3/react-form-builder/pull/43

**Comment by aliyazahid** [comment]
I removed "i" from package.json. Do I need to create PR again ?

**Comment by Kiho** [comment]
Just push fixed code to origin of your PR branch.

**Comment by aliyazahid** [comment]
Yup I pushed the changes please check it.

**Pull request merged**